### PR TITLE
Update SonarQube configuration

### PR DESCRIPTION
**Pull request by Hilbrand**

### CODE REVIEW

File: sonarqube/readme.md [hunk 1]
```diff
@@ -4,59 +4,24 @@ This describes the configuration settings for SonarQube
...
-<module name="SuppressionCommentFilter" />
-```
\ No newline at end of file
+| Rule Key Pattern                         | File Path Pattern                                |
+|------------------------------------------|--------------------------------------------------|
+| `java:S1612`                             | `**/aerius-calculator-wui-client/**/*.java` |
+| `java:S2039`                             | `**/aerius-calculator-wui-client/**/*.java` |
+| `java:S3052`                             | `**/aerius-calculator-wui-client/**/*.java` |
+| `common-java:InsufficientLineCoverage`   | `**/aerius-calculator-wui-client/**/*.java` |
+| `common-java:InsufficientBranchCoverage` | `**/aerius-calculator-wui-client/**/*.java` |
```
Comment (BertScholten) [inline sonarqube/readme.md:27]:
```suggestion
| Rule Key Pattern                         | File Path Pattern                           |
|------------------------------------------|---------------------------------------------|
| `java:S1612`                             | `**/aerius-calculator-wui-client/**/*.java` |
| `java:S2039`                             | `**/aerius-calculator-wui-client/**/*.java` |
| `java:S3052`                             | `**/aerius-calculator-wui-client/**/*.java` |
| `common-java:InsufficientLineCoverage`   | `**/aerius-calculator-wui-client/**/*.java` |
| `common-java:InsufficientBranchCoverage` | `**/aerius-calculator-wui-client/**/*.java` |
```

**Review by BertScholten — COMMENTED** [review]
The quality-profile-aerius.xml is not the one we have defined in sonarcloud? Might be nice to have a 'backup' of that one if possible?

**Comment by Hilbrand** [comment]
The `quality-profile-aerius.xml` is an updated backup from our AERIUS sonar cloud configuration.

**Comment by BertScholten** [comment]
Oh right, I thought the file was deleted, but github just wasn't showing the changes because it was a large diff... Never mind that comment then

**Review by BertScholten — APPROVED** [review]
LGTM, one suggestion for consistency

**Pull request merged**